buildSrc/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+    compile 'com.android.tools.build:gradle:1.2.3'
```

Comment:
Yeah, I know… The only "easy" way I've found: move dependencies to `gradle.properties` in the root of the project (it will be available in the all subprojects but `buildSrc`) and parse this file in the `build.gradle` of `buildSrc`.